### PR TITLE
Beta2 from 50208cc

### PR DESCRIPTION
**Pull request by idiotmax**

### CODE REVIEW

**Comment by Conjuror** [comment]
https://conjuror.github.io/Zerda-Screenshots/index.html for l10n

File: locales/templates/app.pot [hunk 1]
```diff
@@ -441,21 +452,43 @@ msgid "Downloading…"
...
+msgid "Cannot find the file"
 msgstr ""
 
 msgctxt "cannot_delete_the_file"
 msgid "Cannot delete the file."
 msgstr ""
 
+#, c-format
+msgctxt "download_completed"
+msgid "%1$s downloaded"
```
Comment (Delphine) [inline locales/templates/app.pot]: I'd add a comment to explain the variable here

File: locales/templates/app.pot [hunk 2]
```diff
@@ -441,21 +452,43 @@ msgid "Downloading…"
...
+msgstr ""
+
+#, c-format
+msgctxt "download_deleted"
+msgid "%1$s was deleted"
```
Comment (Delphine) [inline locales/templates/app.pot]: Same here, please add a comment to explain variable

File: locales/templates/app.pot [hunk 3]
```diff
@@ -575,35 +608,40 @@ msgstr ""
...
+#, c-format
+msgctxt "screenshot_image_viewer_dialog_info_title1"
+msgid "Title: %1$s"
 msgstr ""
 
-msgctxt "screenshot_image_viewer_dialog_info_url"
-msgid "URL:"
+#, c-format
+msgctxt "screenshot_image_viewer_dialog_info_url1"
+msgid "URL: %1$s"
```
Comment (Delphine) [inline locales/templates/app.pot]: Again, I'd explain de variable to be extra clear.

**Pull request merged**